src/adapters/TelnetAdapter.ts

```diff
@@ -543,13 +543,24 @@ export class TelnetAdapter {
 
                 //write a for loop to print every value from the array. This gets around the `...` after the 100th item issue in the roku print call
             } else if (['roarray', 'rolist', 'roxmllist', 'robytearray'].includes(lowerExpressionType)) {
-                data = await this.requestPipeline.executeCommand(
-                    `for each vscodeLoopItem in ${expression} : print "vscode_is_string:"; (invalid <> GetInterface(vscodeLoopItem, "ifString")); vscodeLoopItem : end for`
-                    , true);
+                const command = [
+                    `for each vscodeLoopItem in ${expression} : print ` +
```

Comment:
I split these into individual lines in our source code for better readability. they are all joined together into a single line still.